modules/apps/emm/client/js/common.js

```diff
@@ -273,7 +273,7 @@ function createFilter(oTable, rowId, selectId, heading){
             var sValue = aData[iColumn];
 
             // ignore empty values?
-            if (bIgnoreEmpty == true && sValue.length == 0) continue;
+            if (bIgnoreEmpty == true && !sValue) continue;
```

Comment:
@milanperera you can just use `bIgnoreEmpty` if it's a boolean type. It will be `if (bIgnoreEmpty && !sValue) continue;`